Cargo.toml

```diff
@@ -13,16 +13,18 @@ crate-type = ["cdylib"]
 test = false
 
 [dependencies]
+anyhow = "1.0.32"
 log = "0.4.8"
 futures = { version = "0.3.4", features = ['async-await'] }
 serde = { version = "1.0.110", features = ["derive"] }
 serde_json = "1.0.53"
-flv-future-aio = { version = "2.2.1" }
-flv-api-sc = { version = "1.2.0"}
 node-bindgen = { version = "2.1.1" }
-kf-protocol = { version = "1.1.2" }
-flv-util = { version = "0.2.0" }
-flv-client = { version = "3.1.0" }
+fluvio = { git = "https://github.com/infinyon/fluvio.git", features =  ["admin"] }
+flv-util = "0.5.1"
+fluvio-types = { version = "0.1.0", git = "https://github.com/infinyon/fluvio.git" }
```

Comment:
Should lock this to specific commit to prevent future breakage.

Reply:
will update dependencies once libraries are published